index.html

```diff
@@ -43,7 +43,7 @@
             </ul>
           </nav>
           <div class="button_block">
-            <button class="submit_button" type="button">
+            <button type="submit">
```

Comment:
nice!

Reply:
but you should move class up instead button_block you should have submit_button :( 

  <div class="submit_button">
            <button type="submit">

Reply:
```
  <div class="submit_block">
            <button type="submit">
```